CMakeLists.txt

```diff
@@ -633,13 +636,15 @@ if(EXECUTORCH_BUILD_PYBIND)
   # TODO(larryliu): Fix macOS 2 dylibs having 2 sets of static variables issue
   if(EXECUTORCH_BUILD_KERNELS_CUSTOM_AOT AND NOT APPLE)
     list(APPEND _dep_libs custom_ops_aot_lib)
+    list(APPEND _dep_libs aqlm)
```

Comment:
Link the custom kernel to the portable_lib library. If custom operators are to be compiled that is.